wye/workshops/models.py

```diff
@@ -51,6 +56,35 @@ class Meta:
     def __str__(self):
         return '{}-{}'.format(self.requester, self.workshop_section)
 
+    def save(self, force_insert=False, force_update=False, using=None):
+        super(Workshop, self).save(force_insert, force_update, using)
```

Comment:
Django uses same save() method for creating and updating object unless explicitly called.
Reference: https://docs.djangoproject.com/en/1.9/ref/models/instances/#what-happens-when-you-save